src/shared/hooks/useRespondentLabel.ts

```diff
@@ -1,7 +1,6 @@
 import { useTranslation } from 'react-i18next';
 
 import { users } from 'redux/modules';
-import { getRespondentName } from 'shared/utils';
 
 export const useRespondentLabel = (isSubject?: boolean) => {
```

Comment:
This hook is used in other places. Could you please check if everything is working properly there?

Reply:
This hook produces the label in Summary and Review tabs at least:
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/16320432/19267f35-1e2a-4d5a-987a-bb1ead8d167f)
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/16320432/a4667d9e-bf53-4e75-a075-232bba4be2fa)